paper-badge.html

```diff
@@ -114,31 +117,35 @@
         this.async(this.notifyResize, 1);
       },
 
+      _forChanged: function() {
+        // The first time the property is set is before the badge is attached,
+        // which means we're not ready to position it yet.
+        if (!this.isAttached) {
+          return;
+        }
+        this.attached();
```

Comment:
It seems a little weird to be calling `attached` manually. Would it be possible to move the relevant behavior to a more appropriately named method?